src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -250,6 +250,15 @@ public final boolean isEmpty() {
     public abstract Builder toBuilder();
 
     /**
+     * Check if this text can be converted to plain and back without loosing any information.
+     *
+     * @return {@code true} if this text is simple.
```

Comment:
I don't think a code tag is needed here, could just say `@return If this text is simple`